docker-compose.yaml

```diff
@@ -271,12 +275,12 @@ services:
       - READY_CHECK_ALL_COMPONENTS=True
       - REDIS_MORPHEUS_TASK_QUEUE=morpheus_task_queue
       # Self-hosted redis endpoints.
-      - YOLOX_GRPC_ENDPOINT=yolox:8001
-      - YOLOX_HTTP_ENDPOINT=http://yolox:8000/v1/infer
-      - YOLOX_INFER_PROTOCOL=grpc
+      - YOLOX_PAGE_ELEMENTS_GRPC_ENDPOINT=yolox-page-elements:8001
+      - YOLOX_PAGE_ELEMENTS_HTTP_ENDPOINT=http://yolox-page-elements:8000/v1/infer
+      - YOLOX_PAGE_ELEMENTS_INFER_PROTOCOL=grpc
```

Comment:
Unfortunately, I think we have to make sure we update all code that makes use of this environment variable if we rename it, e.g.,
https://github.com/NVIDIA/nv-ingest/blob/60be6620de7a98e3e9cc359592bcba4f276e5679/src/nv_ingest/util/pipeline/stage_builders.py#L183
https://github.com/NVIDIA/nv-ingest/blob/60be6620de7a98e3e9cc359592bcba4f276e5679/src/nv_ingest/api/v1/health.py#L67